page-components/tables.md

```diff
@@ -4,6 +4,1217 @@ title:  Tables
 category: Page components
 ---
 
+- [Use cases](#use-cases)
+- [Behavior](#behavior)
+- [Content guidelines](#content-guidelines)
+- [Style](#style)
+- [Code](#code)
+{: class="toc"}
+
+<div class="content-50 content-first">
+
+Tables divide information into distinct columns and rows to create an intersection “cell” where data is displayed.
```

Comment:
We haven't stuck to an 80-character line limit in other Design Manual pages, so I kept long lines in this page, too.